envrc_sample

```diff
@@ -1,5 +1,17 @@
 layout pyenv 3.10.4
 
-dotenv_if_exists
+export AUTHORIZER=casbin_authorizer
+# If you need a specific model file, you can set it here
```

Comment:
should we even mention the file documentation if the ability to use a rbac file doesn't exist yet? or maybe we could put a disclaimer here saying that functionality is coming